nft_ingester/src/program_transformers/bubblegum/delegate.rs

```diff
@@ -30,6 +30,13 @@ where
                 ..
             } => {
                 let id_bytes = id.to_bytes();
+
+                // First check to see if this asset has been decompressed or updated by
+                // `update_metadata`.
```

Comment:
Does update_metadata impact the owner/delegate info? Should this check actually be here?

Reply:
The check didn't need to strictly be there.  The original version 1 for the `!asset_should_be_updated` check was actually just an `asset_was_decompressed` test.  The idea was to just say "if this thing has been decompressed, don't try to do any other compression indexing and just return immediately".

This version 1 check has the same problem with concurrency as pointed out in the other comment, so it won't be there in the next version.

Reply:
Fixed in https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/pull/134